scipy/special/_add_newdocs.py

```diff
@@ -6854,16 +6856,13 @@ def add_newdoc(name, doc):
     where :math:`I` is the regularized incomplete beta function, and
     :math:`x = f d_n/(f d_n + d_d)`.
 
-    The computation time required for this routine is proportional to the
-    noncentrality parameter `nc`.  Very large values of this parameter can
-    consume immense computer resources.  This is why the search range is
-    bounded by 10,000.
+    Note that argument order of `ncfdtr` is different from that of the
+    otherwise equivalent ``cdf`` method of `scipy.stats.ncf`. `f` is the last
```

Comment:
```suggestion
    otherwise equivalent ``cdf`` method of `scipy.stats.ncf`. ``f`` is the last
```

Double back ticks once more, otherwise this will link to the f distribution.

Reply:
I don't think this should be changed.

<img width="709" alt="image" src="https://github.com/user-attachments/assets/fb0ce50b-5a3f-4cfe-a354-a7aea80adebf">

`f` refers to the parameter of this function, and it does not currently link to `scipy.stats.f`.

Reply:
```suggestion
    similar ``cdf`` method of `scipy.stats.ncf`: `f` is the last
```
or "comparable".
There are other differences, like `ncf.cdf` evaluated outside of the support is zero or one whereas the `ncfdtr` function returns NaN.

Reply:
@dschmitz89, can you approve or take back the request so this can be merged without having "requested changes".

Reply:
I can dismiss the review. Looks like everything was addressed.